pynumdiff/utils/evaluate.py

```diff
@@ -118,3 +118,19 @@ def error_correlation(dxdt_hat, dxdt_truth, padding=None):
     r = stats.linregress(dxdt_truth[padding:-padding] -
                                 np.mean(dxdt_truth[padding:-padding]), errors)
     return r.rvalue**2
+
+
+def total_variation(x, padding=0):
```

Comment:
This felt like it belonged here instead of in the `utilities`, since it's only used as part of calculating a metric in the absence of ground truth.